src/main/resources/application.conf

```diff
@@ -43,6 +43,9 @@ backend {
   //  baseExecutionBucket = ""
   //  endpointUrl = ""
   //}
+  local {
+    tryToHardlinkInputs = true
+  }
```

Comment:
I think the 'root' is a different value for JES than it is for the local-filesystem.  In the local-filesystem it represents a directory where cromwell will write things, in JES it represents a bucket.  We'll call `File.mkdirs()` on the `backend.local-filesystem.root` value but we'll do something different with the JES root.